admission-webhook/rockcraft.yaml

```diff
@@ -27,14 +28,23 @@ parts:
       mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
       (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
 
+  # The go plugin uses go install to build and install the "admission-webhook" binary at
+  # /bin/admission-webhook, which is undesirable as it is expected to be in /webhook.
+  # The go plugin does not offer any mechanism for passing custom install paths, so we have
+  # to use override-build instead to make sure the binary is placed in the expected path.
```

Comment:
Could we use `organize` instead of `override-build`?
https://documentation.ubuntu.com/rockcraft/en/stable/common/craft-parts/reference/part_properties/#organize

Reply:
Not really, in my comment I am exposing how the go plugin will install (go install) the binary instead of building it (go build), which will automatically put the binary in /bin/ with a specific name that doesn't seem to be configurable; at the same time `organize` does not seem to be configurable enough to take binaries from a path, only from the cwd, making the whole thing useless.

`override-build` allows us to use go build and know the path of the binary so we can move it around.